cmd/check-imports/main.go

```diff
@@ -0,0 +1,33 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"gonum.org/v1/tools/internal/imports"
+)
+
+var blacklist = []string{
```

Comment:
If were making this publicly useful, this should probably be configurable - maybe a JSON string?

Reply:
done. (I went with just a comma-separated list of regexes.)